CSS/css_completions.py

```diff
@@ -90,8 +102,8 @@ def on_query_completions(self, view, prefix, locations):
             return sublime.CompletionList(items, sublime.INHIBIT_WORD_COMPLETIONS)
         return None
 
-    def complete_property_name(self, view, prefix, pt):
-        text = view.substr(sublime.Region(pt, view.line(pt).end()))
+    def complete_property_name(self, view: sublime.View, prefix: str, pt: sublime_types.Point):
```

Comment:
Return value should be annotated.

Also should change the return statement to returns a list instead of a generator as I don't think it was intentional to return generator here and it also makes types invalid for `sublime.CompletionList` constructor.